dist/tools/pr_check/pr_check.sh

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 SQUASH_COMMITS="$(git log $(git merge-base HEAD "${RIOT_MASTER}")...HEAD --pretty=format:"    %h %s" | \
```

Comment:
You could change the format to `%H %s` to always get the unabbreviated hash instead of relying on some user setting for how long the output should be.

Reply:
But this is used to warn the user/dev afterwards, so it may be a bit to verbose for some people.